src/main/java/io/github/intoto/models/Predicate.java

```diff
@@ -0,0 +1,9 @@
+package io.github.intoto.models;
+
+/**
+ * A generic attestation type with a schema isomorphic to in-toto 0.9. This allows existing in-toto
+ * users to make minimal changes to upgrade to the new attestation format.
+ *
+ * <p>Most users should migrate to a more specific attestation type, such as Provenance.
+ */
+public class Predicate {}
```

Comment:
optional: Have the predicate know it's own type so that users don't have to set it themselves.